changelog/0051-historical-auditcare-data-migration.yml

```diff
@@ -0,0 +1,31 @@
+title: historical-auditcare-data-migration
+key: historical-auditcare-data-migration
+date: 2022-08-11
+optional_per_env: yes
+context: |
+  Instructions regarding Migrating Historical Auditcare data to SQL.
+
+details: |
+  From March 2021, we have started saving audit events to Postgres. The historical data was still in CouchDB. We have added migration utilities which will help you to migrate historical auditcare data. If you do not use auditcare you can safely ignore it.
+
+update_steps: |
+  ### Steps to migrate
+  It is advised to start a tmux session to run the migrations, you can use `django-manage` machine to start the tmux session. 
+    
+  ```
+  $ cchq <env> ssh django-manage:0
+  $ tmux new -s auditcare
+  $ sudo -iu cchq
+  $ cd /home/cchq/www/<env>/current && source python_env/bin/activate
+  $ python manage.py copy_events_to_sql --batch_size=10000 --workers=5
+  ```
```

Comment:
Any reason not to use `django-manage --tmux copy_events_to_sql ...`?

Reply:
I like to give the tmux shell a recognizable name while creating it, it's handy to attach back. It's just a personal preference, I can go with `djnago-manage` if you feel that would be more apt.